app/src/main/java/com/example/internetshop/presentation/activity/fragments/CategoryProductListFragment.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.internetshop.presentation.activity.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.internetshop.databinding.FragmentCategoryProductListBinding
+import com.example.internetshop.model.data.di.component.AppComponent
+import com.example.internetshop.presentation.activity.ContainerHolder
+import com.example.internetshop.presentation.adapters.ProductsCategoryAdapter
+import com.example.internetshop.presentation.adapters.VerticalSpaceItemDecoration
+import com.example.internetshop.presentation.viewModel.CategoryProductListViewModel
+
+class CategoryProductListFragment : BaseFragment() {
+
+    private var binding: FragmentCategoryProductListBinding? = null
```

Comment:
To avoid possible memory leaks, make binding null in `onDestroyView` method

Reply:
fixed